lib/ifelse.dart

```diff
@@ -6,7 +6,7 @@ class Ifelse extends StatelessWidget{
    Widget build(BuildContext context){
     return Scaffold(
       appBar: AppBar(
-        title: Text("if else"),
+        title: Text("if..else                       "),
```

Comment:
Why blank space after if..else ?

Reply:
umm, I think .. in can representation a else if

Reply:
thanks for correction